drivers/pppos.c

```diff
@@ -699,7 +694,7 @@ static int pppos_netifInit(struct netif *netif, char *cfg)
 	memset(state, 0, sizeof(pppos_priv_t));
 	state->netif = netif;
 	state->serialdev_fn = cfg;
-	state->serialat_fn = "/dev/ttyacm1";
+	state->serialat_fn = "/dev/ttyACM0";
```

Comment:
Please do not change hardcoded paths to match your machine.

```suggestion
/* TODO: return error if device path is not found in cfg */
state->serialat_fn = "/dev/ttyacm1";
```